app/ui/style.css

```diff
@@ -1,78 +1,84 @@
 @import url("agregore://theme/vars.css");
 
+:root {
+  --top-height: 2.5em;
+  --border-width: 0.125em;
+}
+
 html, body {
-	width: 100%;
-	height: 100%;
-	padding: 0;
-	margin: 0;
-	display: flex;
-	flex: 1;
-	flex-direction: column;
-	background: var(--ag-color-white);
-	font-family: var(--ag-theme-font-family);
+  width: 100%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  background: var(--ag-color-white);
+  font-family: var(--ag-theme-font-family);
 }
 
 #view {
-	flex:1
+  flex:1
 }
 
-#top {
+#toptop {
```

Comment:
Is this still necessary if the action buttons icons are a fixed height?

Reply:
No, not necessary. I just thought it clean.
Any negatives to doing this? Apart from clutter I suppose?

Reply:
Discussed with @RangerMauve, issue was re:text resize. Tested resizing and it works well.
A reason to keep the height is in case another component has strange heights in future.